src/textord/cjkpitch.cpp

```diff
@@ -733,7 +733,7 @@ bool FPRow::Pass2Analyze() {
         if (!is_final(j)) intersecting = true;
       } else {
         not_intersecting = true;
-        if (i - j > 0) {
```

Comment:
The current code works correct, so there is no need to change it. `j` is initially equal to `i` (so `(i - j) > 0 is false) and then possibly decremented (in that case the test is always true).

Reply:
if these conditions about i and j are true, then everything is really ok. I would appreciate a demonstration of this statement in code.

if it's not time consuming.

Reply:
> I would appreciate a demonstration of this statement in code.

What kind of demonstration are you thinking of?